src/mokka/src/main/java/pl/hycom/mokka/service/openapi/OpenApiParserConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package pl.hycom.mokka.service.openapi;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.ConversionService;
+
+/**
+ * @author Kamil Adamiec (kamil.adamiec@hycom.pl)
+ */
+@Configuration
+public class OpenApiParserConfiguration {
+
+    @Bean
+    public OpenApiParsingService openApiParsingService(ApplicationContext context) {
```

Comment:
`ApplicationContext` not needed. `ConversionService conversionService` should be enough. Spring Boot's magic will do the rest :)